editor/script_create_dialog.cpp

```diff
@@ -787,9 +773,7 @@ ScriptCreateDialog::ScriptCreateDialog() {
 	language_menu = memnew(OptionButton);
 	language_menu->set_custom_minimum_size(Size2(250, 0) * EDSCALE);
 	language_menu->set_h_size_flags(SIZE_EXPAND_FILL);
-	l = memnew(Label(TTR("Language")));
-	l->set_align(Label::ALIGN_RIGHT);
-	gc->add_child(l);
+	gc->add_child(memnew(Label(TTR("Language:"))));
```

Comment:
Is it really useful to add a colon here? I think it's pretty clear without it that the part on the left describes what you fill in the fields on the right.

Reply:
> Made section names be in the same style as other dialogs (starting from the left and ending with a ":").

I just read this, so I guess if it makes things consistent, that's fine. (Note sure that it's my preferred style, but consistency is more important.)